docs/samples/basic_config/key_rotator.yaml

```diff
@@ -0,0 +1,7 @@
+database:
+  # Database URL. (required)
+  url: "postgres://postgres:postgres@localhost:5432/postgres"
+
+key_rotator:
+  # Rotation policy for global HPKE keys
+  hpke: {}
```

Comment:
nit: I think this config is so simple because it is attempting to be as minimal as possible, but I think this would effectively just remove any existing keys.

Maybe specify only `ciphersuite`, with a common choice of configuration values? (`P256HkdfSha256`, `HkdfSha256`, `Aes128Gcm`, perhaps?)

Reply:
An empty `ciphersuite` value should default to one with a reasonable ciphersuite.
```rust
fn default_hpke_ciphersuites() -> HashSet<HpkeCiphersuite> {
    HashSet::from([HpkeCiphersuite::new(
        HpkeKemId::X25519HkdfSha256,
        HpkeKdfId::HkdfSha256,
        HpkeAeadId::Aes128Gcm,
    )])
}
```

At another look I'm not happy with the test coverage on this though, so I'll at minimum add a test.